pkg/devcontainer/run.go

```diff
@@ -91,7 +91,7 @@ type UpOptions struct {
 func (r *runner) Up(ctx context.Context, options UpOptions) (*config.Result, error) {
 	// download workspace source before recreating container
 	_, isDockerDriver := r.Driver.(driver.DockerDriver)
-	if options.Recreate && !isDockerDriver {
+	if (options.Recreate || options.Reset) && !isDockerDriver {
```

Comment:
shouldn't `options.Recreate` already be true because we set it on the command earlier already

Reply:
it was a "just in case" thing but yea it should

Reply:
fixed